Cargo.toml

```diff
@@ -22,6 +9,25 @@ members = [
   "tools/build-wasm-example",
   "errors",
 ]
+[workspace.package]
+version = "0.9.0-dev"
+edition = "2021"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+
+[package]
+name = "bevy"
+version.workspace = true
+edition.workspace = true
+categories = ["game-engines", "graphics", "gui", "rendering"]
```

Comment:
Maybe keep categories for just the main crate? Also the graphics, gui and rendering categories don't apply to most crates. The graphics keyword also doesn't apply to most crates.